usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+                + "    },\n"
+                + "    \"key\": \"1990-01-08T15:20:30-Cloroform85\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"quantity\": 85.0,\n"
+                + "      \"duration\": 45,\n"
+                + "      \"periodicity\": 36\n"
+                + "    },\n"
+                + "    \"key\": \"1995-01-08T15:20:30-Cloroform85\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"quantity\": 85.0,\n"
+                + "      \"duration\": 45,\n"
+                + "      \"periodicity\": 36\n"
+                + "    },\n"
+                + "    \"key\": \"1998-01-08T15:20:30-Cloroform85\"\n"
+                + "  }\n"
+                + "]");
+        medicationDataJson = new StringBuilder("{\n"
+                + "      \"quantity\": 85.0,\n"
+                + "      \"duration\": 45,\n"
+                + "      \"periodicity\": 36\n"
+                + "}"
+        );
+        medicationData = new MedicationData(85.0, 45, 36);
+        medicationCollectionList = new ArrayList<>();
+        medicationCollectionList.add(new Medication(DATE_1, "Cloroform85", medicationData));
+        medicationCollectionList.add(new Medication(DATE_2, "Cloroform85", medicationData));
+        medicationCollectionList.add(new Medication(DATE_3, "Cloroform85", medicationData));
+
+        vaccinationCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"description\": \"It was really pleasant\"\n"
+                + "    },\n"
+                + "    \"key\": \"1990-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"description\": \"It was really pleasant\"\n"
+                + "    },\n"
+                + "    \"key\": \"1995-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"description\": \"It was really pleasant\"\n"
+                + "    },\n"
+                + "    \"key\": \"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        vaccinationDataJson = new StringBuilder("{\n"
+                + "      \"description\": \"It was really pleasant\"\n"
+                + "}"
+        );
+        vaccinationData = new VaccinationData("It was really pleasant");
+        vaccinationCollectionList = new ArrayList<>();
+        vaccinationCollectionList.add(new Vaccination(DATE_1, vaccinationData));
+        vaccinationCollectionList.add(new Vaccination(DATE_2, vaccinationData));
+        vaccinationCollectionList.add(new Vaccination(DATE_3, vaccinationData));
+
+        vetVisitCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"reason\": \"It was really pleasant\",\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "      \"reason\": \"It was really pleasant\",\n" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)